### PR TITLE
Adjust the Missouri deductions metadata to align with model standards

### LINKED ISSUE #4327: Adjust the Missouri deductions metadata to align with model standards
Currently the descriptions and labels need to be changed:
- Descriptions in full sentences 
etc.

### PR DESCRIPTION
**Pull request by Kevintowne**
Fixes #4327

### CODE REVIEW

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: remove all changes related to the expenses

File: policyengine_us/parameters/gov/states/mo/tax/income/deductions/federal_income_tax/ignored_credits.yaml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
+description: Missouri ignores the following credits in federal income tax deduction calculations.
 values:
   2018-01-01:
     - eitc
     - recovery_rebate_credit
 metadata:
   unit: variable
-  name: mo_federal_income_tax_deduction_ignored_credits
-  label: Ignored credits in MO federal income tax deduction
+  label: Missouri ignored credits in federal income tax deduction
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mo/tax/income/deductions/federal_income_tax/ignored_credits.yaml:8]:
```suggestion
  label: Missouri ignored credits in federal income tax deduction
  period: year
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/mo/tax/income/deductions/federal_income_tax/ignored_credits.yaml:8]: add period everywhere, where missing

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/mo/tax/income/credits/property_tax/public_assistance_types.yaml [hunk 2]
```diff
@@ -8,6 +8,7 @@ values:
 metadata:
   unit: currency-USD
   label: Types of public assistance in MO property tax credit gross income
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mo/tax/income/credits/property_tax/public_assistance_types.yaml:10]:
```suggestion
  label: Missouri property tax credit gross income public assistance sources
```

File: policyengine_us/parameters/gov/states/mo/tax/income/credits/property_tax/rent_property_tax_limit.yaml [hunk 3]
```diff
@@ -5,6 +5,7 @@ metadata:
   unit: currency-USD
   period: year
   label: MO property tax credit limit for property renters
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mo/tax/income/credits/property_tax/rent_property_tax_limit.yaml:7]:
```suggestion
  label: Missouri property tax credit limit for property renters
```

File: policyengine_us/parameters/gov/states/mo/tax/income/credits/wftc/eitc_match.yaml [hunk 4]
```diff
@@ -3,7 +3,8 @@ values:
   2023-01-01: 0.1
 metadata:
   unit: /1
-  label: MO EITC percent
+  label: Missouri EITC percent
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mo/tax/income/credits/wftc/eitc_match.yaml:6]:
```suggestion
  label: Missouri EITC match
```

File: policyengine_us/parameters/gov/states/mo/tax/income/deductions/federal_income_tax/ignored_credits.yaml [hunk 5]
```diff
@@ -6,6 +6,7 @@ values:
 metadata:
   unit: variable
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mo/tax/income/deductions/federal_income_tax/ignored_credits.yaml:7]:
```suggestion
  unit: list
```

File: policyengine_us/variables/household/expense/utilities/broadband_cost.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/household/expense/utilities/broadband_cost.py:1]: revert all utility expense related variables - wrong PR

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/mo/tax/income/deductions/mo_max_private_pension.yaml [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
+description: Missouri provides this maximum amount of private pension.
 values:
   2007-01-01: 6_000
 metadata:
   unit: currency-USD
   period: year
-  name: mo_max_private_pension_amount
-  label: MO Max Private Pension Amount
+  label: Missouri maximum private pension amount
+  period: year
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mo/tax/income/deductions/mo_max_private_pension.yaml:8]:
```suggestion
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Comment by MaxGhenis** [comment]
Merging this. Thank you. But let's hold off on further cleanup PRs until merging all possible commits from #4210. I did much of this sort of work there so it would duplicate work and cause merge conflicts.

**Pull request merged**